### ISSUE TITLE
理解学习率调度（余弦退火 + warmup）的原理和作用 | 思想是一个锐化的过程

### ISSUE THREAD

**Issue by TCcjx** [TCcjx/TCcjx.github.io#62]
https://tccjx.github.io/2026-04-23-post29_warmup%E5%92%8C%E5%AD%A6%E4%B9%A0%E7%8E%87%E4%BD%99%E5%BC%A6%E9%80%80%E7%81%AB%E7%9A%84%E9%BB%84%E9%87%91%E7%BB%84%E5%90%88/ 

 1.学习率设置对模型训练的影响 学习率设置过大，会导致训练loss振荡，模型训练不稳定 学习率设置过小，会导致模型训练缓慢 同时，需要注意的是，在模型初期和后期对于学习率的设置是不一样的  2.Warmup阶段在模型刚刚训练的时候，模型的参数都是随机初始化的，如果使用较大的学习率设置，很容易导致模型跑飞。Warmup的思想就是一开始从一个min_learning_rate进行线性增长，增长到目标值
